WatcherBot/appsettings.example.json

```diff
@@ -99,6 +99,10 @@
         "Networking": "Cyan"
       }
     },
+    "KeepAliveThreadIds": [
+      0
+    ],
+    "KeepAliveMessage": "Ah, ha, ha, ha, stayin' alive, stayin' alive",
```

Comment:
Watcher bot would never say this I'm crying right now

Reply:
It's just the example